view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js

```diff
@@ -554,6 +533,36 @@ define(
                     telephone: address.telephone
                 };
             },
+            buildComponentConfiguration: function (paymentMethod, paymentMethodsExtraInfo, result) {
+                var self = this;
+                var email = '';
+                var showPayButton = false;
```

Comment:
this line can be removed and we can hardcode `false` in the configuration object below

Reply:
I think we should keep this for visibility for when we have to add new methods.